R/get_spec_rt.R

```diff
@@ -15,10 +13,12 @@
 #' @param output_status If `FALSE` (default) the calculated rates are returned
```

Comment:
I think using `FALSE` and `TRUE` (i.e. a keyword, rather than value in quotes) implicitly shows that this is evaluated as a Boolean. In theory, the user does not have to supply `FALSE` or `TRUE` they just need to supply a conditional statement that evaluates to `FALSE` or `TRUE`. So technically, not a requiring a Boolean input (and we don't check for type Boolean on this either). This is typical behaviour, I think, in R & Python.